### ISSUE TITLE
Not able to mavenize the Custom XPath Function project

### ISSUE THREAD

**Issue by rokces** [TIBCOSoftware/bw6-plugin-maven#666]
Maven Plugin version: 2.6.0

Maven version: 3.6.3

Product : TIBCO ActiveMatrix Businessworks

Product version: 6.6.0

Component: Either of Maven Eclipse Plugin or Maven build plugin

Steps to reproduce the issue:

I am trying to mavenize the Custom XPath Function project to use it externally in any Tibco BW application project.
I was following the procedure on the page: https://github.com/TIBCOSoftware/bw6-plugin-maven/wiki/Custom-XPath-Function-with-Maven#external-custom-xpath-function-with-maven

First I created a new Plug-in Project and when I right-click it I don't see the option "Generate POM for CXF Project" as it is directed in the Wiki:
![Trying to mavenize](https://user-images.githubusercontent.com/107851180/174594380-a82a2cec-a984-4d58-8cd7-23ed117ecd4b.png)

Am I missing something?

**Comment by shashank-v-kumar** [comment]
Hi @rokces , I'm also facing the same issue. Were you able to resolve it?

**Comment by shashank-v-kumar** [comment]
Hi @rokces, thanks for the reply. I'm also using Tibco BW 6.6.0 and I also think the problem is with this particular version. I was able to mavenize the plugin project by right clicking on the project-> Configure->Convert to Maven Project and install into local maven repo but I'm not able to use it as a maven dependency. The BW application EAR doesn't contain the plugin jar although I've added it as a dependency in the pom.
